applications/NXoptical_spectroscopy.nxdl.xml

```diff
@@ -0,0 +1,1121 @@
+                    be a generic lens or microcope objectives which are used for the
+                    Raman scattering process.
+                </doc>
+                <field name="type">
+                    <enumeration open="true">
+                        <item value="objective"/>
+                        <item value="lens"/>
+                        <item value="glass fiber"/>
+                        <item value="none"/>
+                    </enumeration>
+                </field>
+                <group name="device_information" type="NXfabrication" optional="true"/>
+            </group>
+            <group type="NXwaveplate" optional="true"/>
+            <group type="NXoptical_window" optional="true"/>
+            <group name="polfilter_TYPE" type="NXcomponent" optional="true" nameType="partial">
+                <doc>
+                    polarization filter to prepare light to be measured or to be incident
+                    on the sample.
+                    Generic polarization filter properties may be implemented via NXfilter_pol
+                    at a later stage.
+                </doc>
+                <field name="filter_mechanism" type="NX_CHAR" optional="true">
+                    <doc>
+                        Physical principle of the polarization filter used to create a
+                        defined incident or scattered light state.
+                    </doc>
+                    <enumeration open="true">
+                        <item value="polarization by Fresnel reflection"/>
+                        <item value="birefringent polarizers"/>
+                        <item value="thin film polarizers"/>
+                        <item value="wire-grid polarizers"/>
+                    </enumeration>
+                </field>
+                <field name="specific_polarization_filter_type" type="NX_CHAR" optional="true">
+                    <doc>
+                        Specific name or type of the polarizer used.
+                        
+                        Free text, for example: Glan-Thompson, Glan-Taylor, Rochon Prism, Wollaston
+                        Polarizer...
+                    </doc>
+                </field>
+                <group name="device_information" type="NXfabrication" optional="true"/>
+            </group>
+            <group name="spectralfilter_TYPE" type="NXcomponent" optional="true" nameType="partial">
+                <doc>
+                    Spectral filter used to modify properties of the scattered or incident light.
+                </doc>
+                <field name="filter_type" optional="true">
+                    <doc>
+                        Type of laserline filter used to suppress the laser, if measurements
+                        close to the laserline are performed.
+                    </doc>
+                    <enumeration open="true">
+                        <item value="long-pass filter"/>
```

Comment:
Long/short as in wavelength? Perhaps add to doc.

Reply:
I checked if the implementation in the classes and appdefs that was voted upon resolved this comment - it does.